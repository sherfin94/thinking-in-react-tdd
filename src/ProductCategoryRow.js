import React, {Component} from 'react'

function ProductCategoryRow(props) {
  return(
    <th>
      <td> {props.name} </td>
    </th>
  )
}

module.exports = ProductCategoryRow;
