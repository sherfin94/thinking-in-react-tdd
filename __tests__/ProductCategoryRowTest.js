import React from 'react';
import ProductCategoryRow from '../src/ProductCategoryRow';
import renderer from 'react-test-renderer';

describe("Product category row", function() {

  it(' matches snapshot ', () => {
    const component = renderer.create(
      <ProductCategoryRow name = "Electronics" />
    );
    let json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
