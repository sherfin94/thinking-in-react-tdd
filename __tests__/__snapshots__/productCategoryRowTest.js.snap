exports[`Product category row  matches snapshot  1`] = `
<th>
  <td>
     
    Electronics
     
  </td>
</th>
`;
